Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM.cpp

```diff
@@ -124,7 +124,7 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian(
                 amrex::Real mag_gamma_arrx = MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(1,0,0),mag_gamma_arr);
 
                 // determine if the material is nonmagnetic or not
-                if (mag_Ms_arrx != 0 && mag_alpha_arrx != 0 && mag_gamma_arrx != 0)
+                if (mag_Ms_arrx != 0._rt)
```

Comment:
could Ms ever be < 0?

Reply:
We error check it after we parse it, so no.